cmd/manager/main.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"os"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/klog/klogr"
+	"sigs.k8s.io/cluster-api-provider-docker/api/v1alpha1"
+	"sigs.k8s.io/cluster-api-provider-docker/controllers"
+	ctrl "sigs.k8s.io/controller-runtime"
+	// +kubebuilder:scaffold:imports
```

Comment:
These`kubebuilder:scaffold:*` annotations seem weird, are they just artifacts for the generation or do they serve some other purpose?

Reply:
kubebuilder hooks into this point when generating a new type. kubebuilder automatically inserts imports/init statements etc. for you. This is copied from the kubebuilder book and it seemed safe enough to leave